config/migrations/20220926140003-beleidsvelden-themis-concepts-migratie.sparql

```diff
@@ -0,0 +1,148 @@
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/c08ccb06-460f-4bb7-87d5-714a94c5186b> <http://themis.vlaanderen.be/id/beleidsveld/c08ccb06-460f-4bb7-87d5-714a94c5186b>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/5de93340-ce1a-4886-8425-6c80bbe0351b> <http://themis.vlaanderen.be/id/beleidsveld/5de93340-ce1a-4886-8425-6c80bbe0351b>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/e4dab90c-bb3a-47c5-abfc-802ddb55480d> <http://themis.vlaanderen.be/id/beleidsveld/e4dab90c-bb3a-47c5-abfc-802ddb55480d>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/bc15da36-a7a3-46eb-9af6-d6a8a12b0ad1> <http://themis.vlaanderen.be/id/beleidsveld/bc15da36-a7a3-46eb-9af6-d6a8a12b0ad1>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/13842fbc-441c-46ca-a58d-f78a64eb8a93> <http://themis.vlaanderen.be/id/beleidsveld/13842fbc-441c-46ca-a58d-f78a64eb8a93>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/d5193292-bdc1-4c7d-ba0a-661219108801> <http://themis.vlaanderen.be/id/beleidsveld/d5193292-bdc1-4c7d-ba0a-661219108801>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/8f23ea03-637f-4d4b-8135-7281a6358ab1> <http://themis.vlaanderen.be/id/beleidsveld/8f23ea03-637f-4d4b-8135-7281a6358ab1>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/b5eb0138-b286-473e-8f06-5647db7ac120> <http://themis.vlaanderen.be/id/beleidsveld/b5eb0138-b286-473e-8f06-5647db7ac120>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/a84f868d-de05-4a62-b7eb-fc676479a6d2> <http://themis.vlaanderen.be/id/beleidsveld/a84f868d-de05-4a62-b7eb-fc676479a6d2>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/3ada201d-68dd-41a3-b0da-5a8ef27de879> <http://themis.vlaanderen.be/id/beleidsveld/3ada201d-68dd-41a3-b0da-5a8ef27de879>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/c48135d7-f235-460a-af3b-deb3954e6a32> <http://themis.vlaanderen.be/id/beleidsveld/c48135d7-f235-460a-af3b-deb3954e6a32>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/d1b93810-6066-4b0e-ad31-130c7ae838ac> <http://themis.vlaanderen.be/id/beleidsveld/d1b93810-6066-4b0e-ad31-130c7ae838ac>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/d185e71b-2a6f-49e5-b710-3d9f58ddf11b> <http://themis.vlaanderen.be/id/beleidsveld/d185e71b-2a6f-49e5-b710-3d9f58ddf11b>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/ea4c45da-a07d-49c9-92ab-16d0a0d4e37e> <http://themis.vlaanderen.be/id/beleidsveld/ea4c45da-a07d-49c9-92ab-16d0a0d4e37e>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/43b8de2e-61f9-4cf8-a018-9f3cc27f4103> <http://themis.vlaanderen.be/id/beleidsveld/43b8de2e-61f9-4cf8-a018-9f3cc27f4103>)
+    (<http://kanselarij.vo.data.gift/id/beleidsveld/2b0103aa-a3e8-4718-80f8-8de405966c57> <http://themis.vlaanderen.be/id/beleidsveld/2b0103aa-a3e8-4718-80f8-8de405966c57>)
+  }
+  GRAPH ?g {
+    ?oldGovernmentField ?p ?o .
+    ?oldGovernmentField skos:prefLabel ?oldLabel .
+    FILTER (?p != skos:prefLabel)
```

Comment:
Same thing here. Won't this overwrite `mu:uuid`, `prov:generatedAtTime`, `schema:position`, `skos:broader`,  `skos:inScheme` and `skos:topConceptOf` for the newly added fields in the previous migration?